object/asset.go

```diff
@@ -130,6 +131,24 @@ func GetAsset(id string) (*Asset, error) {
 	return getAsset(owner, name)
 }
 
+func GetAssetByName(name string) (*Asset, error) {
```

Comment:
Use `GetAsset()`, don't create new API, the owner param is the Casdoor org, you can get it